pkg/logger/logger.go

```diff
@@ -36,6 +37,17 @@ func InitLog(log *logr.Logger) {
 	Log = *log
 }
 
+func getGoroutineID() int {
```

Comment:
It seems Go team discourages this way for poor performance.
Go standard library’s internal uses (like in the http/2 package) only employ this technique for debugging purposes.